execution.bs

```diff
@@ -4108,39 +4163,235 @@ namespace std::execution {
+    those results. A sender is either valid or invalid.
+
+10. A <dfn export=true>scheduler</dfn> is an abstraction of an execution resource
+    with a uniform, generic interface for scheduling work onto that resource. It
+    is a factory for senders whose asynchronous operations execute value
+    completions on an execution agent belonging to the scheduler's associated
+    execution resource. A <dfn export=true>schedule-expression</dfn> obtains
+    such a sender from a scheduler. A <dfn export=true>schedule sender</dfn> is
+    the result of a schedule expression. A schedule sender produces asynchronous
+    operation(s) that, on success, execute a value completion with an empty set
+    of value results. Multiple schedulers can refer to the same execution
+    resource. A scheduler can be valid or invalid. A scheduler becomes invalid
+    when the execution resource to which it refers becomes invalid, as do any
+    schedule senders obtained from the scheduler, and any operation states
+    obtained from those senders.
+
+11. An asynchronous operation has one or more associated completion schedulers. A
+    <dfn export=true>completion scheduler</dfn> of an asynchronous operation is
+    a scheduler whose associated execution resource is used to execute a
+    completion operation for the asynchronous operation. A value completion
+    scheduler is a scheduler on which an asynchronous operation's value
+    completion operation can execute. Likewise for error completion schedulers
+    and stopped completion schedulers.
+
+12. A sender has an associated queryable object known as its <dfn
+    export=true>attributes</dfn> that describe various characteristics of the
+    sender and of the asynchronous operation(s) it produces. A sender can
+    expose the completion scheduler(s) of its asynchronous operations via
+    its attributes. If present, a value completion scheduler attribute
+    indicates that the sender's asynchronous operations only execute value
+    completion operations on that scheduler's associated execution resource.
+    Likewise for error and stopped completion scheduler attributes.
+    A schedule sender is required to have a value completion scheduler attribute
+    whose value is equal to the scheduler that produced the schedule sender.
+
+13. A <dfn export=true>completion signature</dfn> is a function type that
+    describes a completion operation. An asychronous operation has a finite set
+    of possible completion signatures. The completion signature's return type is
+    the completion tag associated with the completion function that executes the
+    completion operation. The completion signature's argument types are the
+    types and value categories of the asynchronous operation's result datums,
+    determined as if by `decltype`. Together, a sender type and an environment
+    type `E` uniquely determine the set of completion signatures of the
```

Comment:
> Together, a sender type and an environment type `E` uniquely determine the set of completion signatures

I don't think "uniquely" is the right term here?
There might be other such sender / env types that produce the same set of completion signatures.

Maybe you mean "solely determine" or "completely determine" or just simply "determine"?